build.sh

```diff
@@ -0,0 +1,7 @@
+rm -r build
+mkdir build
+cd build
+cmake ..
+make
```

Comment:
Do you think having `cmake .. && make` makes more sense? If any one of them fails, then the other should ideally not run as well.

Reply:
Yes. I have updated this in next commit.